### ISSUE TITLE
[FileNotFoundError] CS_train.py model can not run.

### ISSUE THREAD

**Issue by hjr1391** [nokpil/AgentNet#1]
Dear,
When running the CS_train.py, 
Error ups,
![lack of file2](https://user-images.githubusercontent.com/68892943/99137613-073f2d80-2667-11eb-99d5-b7d886a0bd13.png)

![lack of file](https://user-images.githubusercontent.com/68892943/99137400-829fdf80-2665-11eb-8658-a9332c646a04.png)

My Dependencies:
torch: 1.7.0
python:3.8

Is there a missing file on this model? Or shall l create this file by myself? How to create it?

How to solve the question?
Could U help me ,please ?
Thx

**Comment by nokpil** [comment]
Thanks for your concern. 

Nothing is wrong with your environment, and we hadn't uploaded the dataset and generation file yet. Since the CS dataset is greater than a gigabyte, it is hard to upload it directly. (If there is a way to bypass the capacity cap of GitHub, let me know.)
All of the CS data can be directly processed from the original dataset from [Three-dimensional trajectories and network analyses of group behavior within chimney swift flocks during approaches to the roost] (https://royalsocietypublishing.org/doi/10.1098/rspb.2016.2602#d3e1386), where its dataset is deposited at Dryad (https://datadryad.org/stash/dataset/doi:10.5061/dryad.p68f8).  We are still working on connecting the pipeline for data generation (especially for CS due to the complex inspection scheme) since it is separated into various parts in our original code. 

We'll keep updating this repo. Again, thanks for your attention.

**Comment by hjr1391** [comment]
If you want to try to upload Large File, you can try this (https://git-lfs.github.com/).  I hope those useful for you.
What' s more. After l have tried the other model in train.py, it has the same problem. Does it mean that All model framework but the data has been fully uploaded in this repo?
Thank you!

**Comment by nokpil** [comment]
Yes, sorry for the late response. Currently, the team is waiting for the peer review and focusing on other projects. We'll check the link you've provided. Also, we'll upload the code for data generation that we used for the main paper. Thanks for your attention.
